js/pages/options.es6.js

```diff
@@ -14,6 +15,7 @@ function Options (ops) {
 
 Options.prototype = $.extend({},
     Parent.prototype,
+    mixins.setBrowserClassOnBodyTag,
```

Comment:
Twofold:
- A mix-in can be shared between different pages
- i want this to be the definitive way to do it (and place to do it in). So by putting the mixin inside of `js/pages/mixins`, I'm basically saying, only do use this at the `js/pages` level (instead of sprinkling it around codebase wherever).